containers/covabot/src/cova-bot/covaBot.ts

```
@@ -0,0 +1,307 @@
+					const responseText = await trigger.response(mockMessage as unknown as Message);
+					if (!responseText) {
+						logger.debug('[CovaBot] Empty response from trigger');
+						continue;
+					}
+
+					logger.debug(`[CovaBot] Web response generated: "${responseText}"`);
+					return responseText;
+
+				} catch (error) {
+					logger.error(`[CovaBot] Error in trigger "${trigger.name}":`, error as Error);
+					continue; // Try next trigger
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)